src/components/dialogs/network-modifications/line-split-with-voltage-level/line-split-with-voltage-level-illustration.tsx

```diff
@@ -85,30 +84,29 @@ const LineSplitWithVoltageLevelIllustration = () => {
          * @param svgElt : element fetched from the svg that should be updated
          */
         function updateOpacity(propStr: string, svgElt: HTMLElement | null) {
-            if (svgElt != null) {
-                const indexOpacity = svgElt.style.cssText.indexOf(propStr);
+            if (!!svgElt) {
+                const eltCssText = svgElt.style.cssText;
```

Comment:
we need eltCssText only inside the 'if' block (  if (indexOpacity !== -1) {)

Reply:
I use it before the block ``` if (indexOpacity !== -1) {)``` because I think it makes ```const indexOpacity = eltCssText.indexOf(propStr);``` easier to read.

(instead of ```const indexOpacity = svgElt.style.cssText.indexOf(propStr);``` )

Reply:
if it is easier to read, then it is ok